doc/api/fs.md

```diff
@@ -3918,6 +3918,33 @@ On Linux, positional writes do not work when the file is opened in append mode.
 The kernel ignores the position argument and always appends the data to
 the end of the file.
 
+#### filehandle.write(string[, position[, encoding]])
+<!-- YAML
+added: v10.0.0
+-->
+
+* `string` {string}
+* `position` {integer}
+* `encoding` {string} **Default:** `'utf8'`
+* Returns: {Promise}
+
+Write `string` to the file. If `string` is not a string, then
+the value will be coerced to one.
+
```

Comment:
Misses the information about returned value in the resulting `Promise`.
See https://github.com/nodejs/node/blob/master/doc/api/fs.md#filehandlewritebuffer-offset-length-position for comparison:
> The `Promise` is resolved with an object containing…

Reply:
Otherwise, looks good to me.